README.md

```diff
@@ -157,22 +157,18 @@ Options:
           Set host IP to which the API server will bind [env: HOPRD_API_HOST=]
       --apiPort <PORT>
           Set port to which the API server will bind [env: HOPRD_API_PORT=]
-      --defaultSessionListenHost <DEFAULT_SESSION_LISTEN_HOST>
-          Default Session listening host for Session IP forwarding [env: HOPRD_DEFAULT_SESSION_LISTEN_HOST=]
       --apiToken <TOKEN>
           A REST API token and for user authentication [env: HOPRD_API_TOKEN=]
       --password <PASSWORD>
           A password to encrypt your keys [env: HOPRD_PASSWORD=]
-      --provider <PROVIDER>
-          A custom provider to be used for the node to connect to blockchain [env: HOPRD_PROVIDER=]
+      --blokliUrl <URL>
+          URL for Blokli provider to be used for the node to connect to blockchain [env: HOPRD_BLOKLI_URL=]
```

Comment:
There are more `--provider` references below this line in the example executions.